### ISSUE TITLE
Suggestions for backend

### ISSUE THREAD

**Issue by Phippsy** [MarkEdmondson1234/antiFakeNews#1]
- Reputation metrics: maintain a 'blacklist' of known debunked and/or validated articles to influence reputation?
- Contribute to publisher / sharer categorisations using [tf-idf ](https://en.wikipedia.org/wiki/Tf%E2%80%93idf) on entities to identify and quantify high-affinity topics. Use 'publisher' as the defined collection for tf-idf.
- Surface 'most reputable sources by entity' for any given user-selected entity, to help people find the most trustworthy source?

**Comment by MarkEdmondson1234** [comment]
>     Reputation metrics: maintain a 'blacklist' of known debunked and/or validated articles to influence reputation?

Yes, a liberal hippy may get very low reputation scores for news sources, but a right wing fascist could get very high scores for the same.  Everyone will have a chance to input a whitelist and blacklist of articles they disagree with, and then through network associations with who share or link to those lists, build up a rank of new sources they come across.  Hopefully also update those scores as they go.

> Contribute to publisher / sharer categorisations using tf-idf on entities to identify and quantify high-affinity topics. Use 'publisher' as the defined collection for tf-idf.

This sounds similar to the entity score "salience" in the API responses got so far?

> Surface 'most reputable sources by entity' for any given user-selected entity, to help people find the most trustworthy source?

Cool, yes.
